check-osp/osp_checks.py

```diff
@@ -114,15 +114,23 @@ def check_services_status(system, **kwargs):
             continue
         ssh = ssh_client(host, username="root", password=system.password)
         with ssh:
-            for service_name, status in services.iteritems():
-                service_status = is_service_in_status(ssh, service_name, status)
+            service_status_dict = get_services_status_list(ssh)
```

Comment:
maybe not much difference but I feel that line should be before `with ssh`. As you pass `ssh` to the function already in context manager, what is the purpose of it?

Reply:
Fixed that.